media/static-bundles.json

```diff
@@ -300,9 +300,15 @@
     },
     {
       "files": [
-        "css/foundation/annual-report-2019.scss"
+        "css/foundation/annual-report-2019-2020.scss"
       ],
       "name": "annual_report_2019"
+    },
+     {
+      "files": [
+        "css/foundation/annual-report-2019-2020.scss"
+      ],
+      "name": "annual_report_2020"
```

Comment:
If the 2019 and 2020 bundles are identical they really don't need to be two bundles. You could rename it `annual_report_2019_2020` if you want to be consistent.